### ISSUE TITLE
iam: Problem with AWSLambdaExecute permission

### ISSUE THREAD

**Issue by DariuszWietecha** [localstack/localstack#2274]
# Type of request: This is a ...

[x ] bug report
[ ] feature request

# Detailed description
I can't add  AWSLambdaExecute and AWSXRayDaemonWriteAccess permission to the IAM role. 
I'm on this step of 'Using AWS Lambda with Amazon S3' tutorial:
https://docs.aws.amazon.com/lambda/latest/dg/with-s3-example.html#with-s3-create-execution-role
Am I doing something wrong?

## Expected behavior

Permission will be added

## Actual behavior

Error:
`Unable to parse response (syntax error: line 1, column 54), invalid XML received:
b'<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">\n<title>500 Internal Server Error</title>\n<h1>Internal Server Error</h1>\n<p>The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.</p>\n'`

# Steps to reproduce
1. Create role:
`aws iam create-role --role-name lambda-s3-role --assume-role-policy-document file://trust-policy.json`

trust-policy.json
```
{
  "Role": {
    "Path": "/",
    "RoleName": "lambda-s3-role",
    "RoleId": "AROAQFOXMPL6TZ6ITKWND",
    "Arn": "arn:aws:iam::123456789012:role/lambda-s3-role",
    "CreateDate": "2020-01-17T23:19:12Z",
    "AssumeRolePolicyDocument": {
      "Version": "2012-10-17",
      "Statement": [
        {
          "Effect": "Allow",
          "Principal": {
            "Service": "lambda.amazonaws.com"
          },
          "Action": "sts:AssumeRole"
        }
      ]
    }
  }
}
```

2. Add AWSLambdaExecute permissions to the role
`aws --endpoint-url=http://192.168.99.100:4566 iam attach-role-policy --role-name lambda-s3-role --policy-arn arn:aws:iam::aws:policy/service-role/AWSLambdaExecute`

## Command used to start LocalStack
Localstack v0.11.0

```
$ docker images --digests
REPOSITORY                                    TAG                 DIGEST                                                                    IMAGE ID            CREATED             SIZE
localstack/localstack                         latest              sha256:193557bae30f0c906f3aed3093f61ea214a57b1be12607c539779f605cc9ef01   8e152cf7d57f        47 hours ago        948MB
```

docker-compose.yml:
```
version: '2.1'
services:
  localstack:
    privileged: true
    image: localstack/localstack
    ports:
      - "4566-4599:4566-4599"
      - "${PORT_WEB_UI-8080}:${PORT_WEB_UI-8080}"
    environment:
      - SERVICES=${SERVICES- }
      - DEBUG=${DEBUG- }
      - DATA_DIR=${DATA_DIR- }
      - PORT_WEB_UI=${PORT_WEB_UI- }
      - LAMBDA_EXECUTOR=docker
      - KINESIS_ERROR_PROBABILITY=${KINESIS_ERROR_PROBABILITY- }
      - DOCKER_HOST=unix:///var/run/docker.sock
    volumes:
      - "${TMPDIR:-/tmp/localstack}:/tmp/localstack"
      - /var/run/docker.sock:/var/run/docker.sock
```

and command:
`docker-compose -f ./docker-compose.yml up -d`

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)
Described in the 'Steps to reproduce' section

**Comment by whummer** [comment]
Thanks for reporting @DariuszWietecha . This should be fixed in #2289 - can you please give it a try with the `latest` version? Please report here if the problem persists. Thanks

**Comment by syna-tr** [comment]
Hi @whummer,

I'm getting the same error today.

docker images --digests
REPOSITORY                          TAG                 DIGEST                                                                    IMAGE ID            CREATED             SIZE
localstack/localstack               latest              sha256:99375bba463313d33e784eaa0ab6a20de7469caf530b788e626fbc0e4e90bd17   6ec3db68b985        8 hours ago         998MB
